client/tests/input_selection/alias_outputs.rs

```diff
@@ -2040,3 +2040,47 @@ fn do_not_change_amount_of_governance_transition() {
         })
     ));
 }
+
+#[test]
+fn state_transition_required_but_state_controller_not_provided() {
+    let protocol_parameters = protocol_parameters();
+    let alias_id_1 = AliasId::from_str(ALIAS_ID_1).unwrap();
+
+    let inputs = build_inputs(vec![Alias(
+        2_000_000,
+        alias_id_1,
+        0,
+        BECH32_ADDRESS_ED25519_0,
+        BECH32_ADDRESS_ED25519_1,
+        None,
+        None,
+        None,
+        None,
+    )]);
+    let outputs = build_outputs(vec![Basic(
+        1_000_000,
+        BECH32_ADDRESS_ED25519_0,
+        None,
+        None,
+        None,
+        None,
+        None,
+        None,
+    )]);
+
+    let selected = InputSelection::new(
+        inputs,
+        outputs,
+        addresses(vec![BECH32_ADDRESS_ED25519_1, BECH32_ADDRESS_ED25519_2]),
```

Comment:
```suggestion
        addresses(vec![BECH32_ADDRESS_ED25519_1]),
```
useless?

Reply:
Yeah, kind of a remnant from another test